src/example-types/metricGroups/pivot.js

```diff
@@ -124,16 +191,73 @@ let getSortField = ({ columnValues = [], valueProp, valueIndex } = {}) =>
     valueProp,
   ])
 
+let mapDrilldownPages = node => {
+  let pagination = node.pagination || { columns: {}, rows: {} }
+  let isDrilldown = _.flow(
+    _.flatMapDeep(type =>
+      _.map(prop => _.get([type, prop], pagination), ['drilldown', 'skip'])
+    ),
+    _.some(_.negate(_.isEmpty))
+  )(['columns', 'rows'])
```

Comment:
same as above, this might be trying too hard to be clever

Reply:
Addressed